pkg/checks/checks.go

```diff
@@ -32,6 +32,9 @@ func WaitForHTTPStatusOk(ctx context.Context, transport http.RoundTripper, urlto
 		resp, err := cli.Do(req)
 		if err, ok := err.(*url.Error); ok {
 			if err, ok := err.Err.(*net.OpError); ok {
+				if _, ok := err.Err.(*net.DNSError); ok {
+					return false, nil
```

Comment:
Is this valid? I think @jim-minter wants DNS errors to be permanent failures (see https://github.com/openshift/openshift-azure/pull/131)

Reply:
I think chose to merge https://github.com/openshift/openshift-azure/pull/131 we good. Need to test with POD approach, but at the moment it fails 100% of times when using a new dns approach because of DNS creation race condition. So it needs to be handled for sure. Question how.